tests/Data/Test_DSPowSolution.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2019 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <string>
+#include <vector>
+#include "common/Constants.h"
+#include "libData/MiningData/DSPowSolution.h"
+#include "libTestUtils/TestUtils.h"
+#include "libUtils/Logger.h"
+
+#define BOOST_TEST_MODULE dspowsolutiontest
+#define BOOST_TEST_DYN_LINK
+#include <boost/test/unit_test.hpp>
+
+using namespace boost::multiprecision;
+using namespace std;
+
+BOOST_AUTO_TEST_SUITE(dspowsolutiontest)
+
+BOOST_AUTO_TEST_CASE(testDSPowSolutionClass) {
+  INIT_STDOUT_LOGGER();
+  LOG_MARKER();
+
+  // Satisfy linker
+  Account acc1(0, 0);
```

Comment:
This variable is useless

Reply:
It seems like this was added to fix a linker issue with Messenger. I think we can keep it in the meantime.

Reply:
Unfortunately @ansnunez is right - I was not able to find any appropriate solution for linker satisfaction - still not clear to me why this is happening.

Reply:
I see